src/client/Microsoft.Identity.Client/Http/HttpManagerFactory.cs

```diff
@@ -14,16 +14,19 @@ namespace Microsoft.Identity.Client.Http
     /// </summary>
     internal sealed class HttpManagerFactory
     {
-        public static IHttpManager GetHttpManager(IMsalHttpClientFactory httpClientFactory, bool withRetry, bool isManagedIdentity)
+        public static IHttpManager GetHttpManager(
+            IMsalHttpClientFactory httpClientFactory,
+            bool withRetry,
+            bool isManagedIdentity)
         {
             if (!withRetry)
             {
-                return new HttpManager(httpClientFactory);
+                return new HttpManager(httpClientFactory, HttpRetryConditions.NoRetry);
```

Comment:
This is ok for now, but i expect there will be a need for more granularity in the future and this Rerty will be placed on `HttpManager.MakeHttpCall` method.